<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NhanVienDialog</class>
 <widget class="QDialog" name="NhanVienDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabNhanVien">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <property name="tabsClosable">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Thông tin</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>Tên</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Quê</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ngày/tháng/năm sinh</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Dân tộc</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Vị trí công việc</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>37</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>MSNV</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Ten">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Que">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>151</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Sinh">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_DanToc">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_Mail">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_VTCT">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>130</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_MSNV">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Thông báo</string>
    </attribute>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>661</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>240</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Update thông báo</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <attribute name="title">
     <string>Request</string>
    </attribute>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>69</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Employee ID:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>61</y>
       <width>38</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Reason</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSendLeaveRequest">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>228</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Gửi yêu cầu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="employeeIdLineEdit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>31</y>
       <width>108</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="reasonTextEdit">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>83</y>
       <width>549</width>
       <height>139</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
